tests/provision_test.go

```diff
@@ -190,12 +190,26 @@ func leakProtectionTest() {
 	By("Deleting storage class", deleteStorageClass)
 }
 
+// This acts as a test just to call the new wrapper functions,
+// Once we write tests calling them this will not be required.
+// This doesnt test any Openebs component.
+func lvmOps() {
```

Comment:
Why is this required? Isn't there a script which sets these up already?

Reply:
No that will be in next PR.